doitlive.py

```diff
@@ -22,9 +22,12 @@
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
+from termutils import raw_mode
 
 import click
-from click import echo, style, secho, getchar
+click.disable_unicode_literals_warning = True  # this fixes #12
```

Comment:
I know, it probably shouldn't be a long term fix. We could also check the click version and then set it depending on whether warning messages will appear.